app/main.py

```diff
@@ -1,2 +1,130 @@
+import copy
+
+
+class DictionaryMember:
+    def __init__(
+            self,
+            key: int | str | tuple | float,
```

Comment:
You can use `Hashable` as type hint
```suggestion
            key: Hashable,
```